include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2140,8 +2140,8 @@ namespace internal
         const DoFCellAccessor<dim, spacedim, level_dof_access> &accessor)
       {
         if (accessor.dof_handler->hp_capability_enabled == false)
-          return 0; // ::DoFHandler only supports a single active FE with index
-                    // zero
+          // ::DoFHandler only supports a single active FE with index zero
+          return DoFHandler<dim, spacedim>::default_fe_index;
```

Comment:
The comment is no longer correct. It was about a time when there were `::DoFHandler` and `hp::DoFHandler`. Now there is only one. The comment can simply be removed.